index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Document</title>
+  <link rel="stylesheet" href="css/style.css" />
+  <script src="https://kit.fontawesome.com/5784b9ea6f.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <div class="wrapper">
+    <header>
+      <nav class="flex-nav">
+        <h1 class="logo"><a href="#">My Logo</a></h1>
+        <div class="toggleNav"><i class="fa fa-bars"></i></div>
+        <ul>
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </nav>
+    </header>
+    <section>
+      <h1>I&rsquo;m Francis.<br />Glad to see you!</h1>
+      <p>
+        I&rsquo;m a software developer! I can help you build a product ,
+        feature or website Look through some of my work and experience! If you
+        like what you see and have a project you need coded, don&rsquo;t
+        hesitate to contact me.
+      </p>
+      <a href="#" class="connect">LET’S CONNECT</a>
+      <ul class="social">
+        <li>
+          <a href="#"><i class="fa-brands fa-twitter"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa-brands fa-linkedin"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa-brands fa-medium"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa-brands fa-github"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa-brands fa-angellist"></i></a>
+        </li>
+      </ul>
```

Comment:
Nice job using flexbox to layout your social links👍. Let's try to make sure it sticks to the design by adjusting the space (margin and padding) around them😉.

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img width="300" alt="image" src="https://github.com/lukademinnosa/module1_project1/assets/64643995/52065b6d-7978-4b02-a931-db4833815678">|<img width="300" alt="image" src="https://github.com/lukademinnosa/module1_project1/assets/64643995/51e38bd5-cb3e-42e1-9633-95008a625fa2">|